<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1920</width>
    <height>1080</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>Arial</family>
    <pointsize>10</pointsize>
   </font>
  </property>
  <property name="contextMenuPolicy">
   <enum>Qt::DefaultContextMenu</enum>
  </property>
  <property name="windowTitle">
   <string>DSC Application</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QDockWidget" name="loggerDockWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>720</y>
      <width>1200</width>
      <height>281</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="contextMenuPolicy">
     <enum>Qt::NoContextMenu</enum>
    </property>
    <property name="windowTitle">
     <string>Logger</string>
    </property>
    <widget class="QWidget" name="dockWidgetContents_2">
     <layout class="QGridLayout" name="gridLayout_3">
      <item row="0" column="0">
       <widget class="QTableWidget" name="loggerTable">
        <property name="font">
         <font>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="editTriggers">
         <set>QAbstractItemView::NoEditTriggers</set>
        </property>
        <property name="rowCount">
         <number>0</number>
        </property>
        <property name="columnCount">
         <number>4</number>
        </property>
        <attribute name="verticalHeaderVisible">
         <bool>false</bool>
        </attribute>
        <column>
         <property name="text">
          <string>Time</string>
         </property>
         <property name="font">
          <font>
           <family>Arial</family>
           <pointsize>12</pointsize>
          </font>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Source</string>
         </property>
         <property name="font">
          <font>
           <family>Arial</family>
           <pointsize>12</pointsize>
          </font>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Severity</string>
         </property>
         <property name="font">
          <font>
           <family>Arial</family>
           <pointsize>12</pointsize>
          </font>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Log</string>
         </property>
         <property name="font">
          <font>
           <family>Arial</family>
           <pointsize>12</pointsize>
          </font>
         </property>
        </column>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QTableWidget" name="footerDataTable">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>1000</y>
      <width>1901</width>
      <height>21</height>
     </rect>
    </property>
    <property name="showGrid">
     <bool>false</bool>
    </property>
    <property name="rowCount">
     <number>0</number>
    </property>
    <property name="columnCount">
     <number>10</number>
    </property>
    <attribute name="horizontalHeaderVisible">
     <bool>false</bool>
    </attribute>
    <attribute name="verticalHeaderVisible">
     <bool>false</bool>
    </attribute>
    <column/>
    <column/>
    <column/>
    <column/>
    <column/>
    <column/>
    <column/>
    <column/>
    <column/>
    <column/>
   </widget>
   <widget class="QDockWidget" name="faultsDockWidget">
    <property name="geometry">
     <rect>
      <x>1210</x>
      <y>720</y>
      <width>361</width>
      <height>281</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="contextMenuPolicy">
     <enum>Qt::NoContextMenu</enum>
    </property>
    <property name="windowTitle">
     <string>Active Faults</string>
    </property>
    <widget class="QWidget" name="dockWidgetContents_3">
     <layout class="QGridLayout" name="gridLayout_4">
      <item row="0" column="0">
       <widget class="QTableWidget" name="faultsTable">
        <property name="font">
         <font>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="editTriggers">
         <set>QAbstractItemView::NoEditTriggers</set>
        </property>
        <property name="rowCount">
         <number>0</number>
        </property>
        <property name="columnCount">
         <number>4</number>
        </property>
        <attribute name="verticalHeaderVisible">
         <bool>false</bool>
        </attribute>
        <column>
         <property name="text">
          <string>ID</string>
         </property>
         <property name="font">
          <font>
           <family>Arial</family>
           <pointsize>12</pointsize>
          </font>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Name</string>
         </property>
         <property name="font">
          <font>
           <family>Arial</family>
           <pointsize>12</pointsize>
          </font>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Unit</string>
         </property>
         <property name="font">
          <font>
           <family>Arial</family>
           <pointsize>12</pointsize>
          </font>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Subunit</string>
         </property>
         <property name="font">
          <font>
           <family>Arial</family>
           <pointsize>12</pointsize>
          </font>
         </property>
        </column>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QDockWidget" name="applicationDockWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1920</width>
      <height>719</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="contextMenuPolicy">
     <enum>Qt::NoContextMenu</enum>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="windowTitle">
     <string>Application</string>
    </property>
    <widget class="QWidget" name="dockWidgetContents_4">
     <layout class="QGridLayout" name="gridLayout_5">
      <item row="0" column="0">
       <widget class="QTabWidget" name="applicationTabWidget">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="tabPosition">
         <enum>QTabWidget::North</enum>
        </property>
        <property name="currentIndex">
         <number>0</number>
        </property>
        <property name="tabsClosable">
         <bool>true</bool>
        </property>
        <widget class="QWidget" name="tabAutodetection">
         <attribute name="title">
          <string>Autodetection</string>
         </attribute>
         <layout class="QGridLayout" name="gridLayout">
          <item row="0" column="0">
           <widget class="QFrame" name="frame">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>100</height>
             </size>
            </property>
            <property name="frameShape">
             <enum>QFrame::NoFrame</enum>
            </property>
            <property name="frameShadow">
             <enum>QFrame::Plain</enum>
            </property>
            <property name="lineWidth">
             <number>3</number>
            </property>
            <widget class="QFrame" name="frame_2">
             <property name="geometry">
              <rect>
               <x>9</x>
               <y>9</y>
               <width>930</width>
               <height>615</height>
              </rect>
             </property>
             <property name="minimumSize">
              <size>
               <width>0</width>
               <height>100</height>
              </size>
             </property>
             <property name="frameShape">
              <enum>QFrame::Box</enum>
             </property>
             <property name="frameShadow">
              <enum>QFrame::Plain</enum>
             </property>
             <property name="lineWidth">
              <number>3</number>
             </property>
             <widget class="QLabel" name="label">
              <property name="geometry">
               <rect>
                <x>10</x>
                <y>30</y>
                <width>210</width>
                <height>140</height>
               </rect>
              </property>
              <property name="text">
               <string/>
              </property>
              <property name="pixmap">
               <pixmap resource="qdarkstyle/style.qrc">:/qss_icons/Images/ADS1248.gif</pixmap>
              </property>
             </widget>
             <widget class="QLabel" name="label_2">
              <property name="geometry">
               <rect>
                <x>10</x>
                <y>220</y>
                <width>210</width>
                <height>140</height>
               </rect>
              </property>
              <property name="text">
               <string/>
              </property>
              <property name="pixmap">
               <pixmap resource="qdarkstyle/style.qrc">:/qss_icons/Images/DRV593.gif</pixmap>
              </property>
             </widget>
             <widget class="QLabel" name="label_4">
              <property name="geometry">
               <rect>
                <x>10</x>
                <y>410</y>
                <width>210</width>
                <height>140</height>
               </rect>
              </property>
              <property name="text">
               <string/>
              </property>
              <property name="pixmap">
               <pixmap resource="qdarkstyle/style.qrc">:/qss_icons/Images/LMP90100.gif</pixmap>
              </property>
             </widget>
             <widget class="QLabel" name="labelStatusAds1248Info1">
              <property name="geometry">
               <rect>
                <x>240</x>
                <y>50</y>
                <width>200</width>
                <height>41</height>
               </rect>
              </property>
              <property name="font">
               <font>
                <pointsize>18</pointsize>
                <weight>50</weight>
                <bold>false</bold>
               </font>
              </property>
              <property name="text">
               <string>ADS1248</string>
              </property>
             </widget>
             <widget class="QLabel" name="labelStatusAds1248Info2">
              <property name="geometry">
               <rect>
                <x>240</x>
                <y>80</y>
                <width>111</width>
                <height>41</height>
               </rect>
              </property>
              <property name="font">
               <font>
                <pointsize>18</pointsize>
                <weight>50</weight>
                <bold>false</bold>
               </font>
              </property>
              <property name="text">
               <string>Status:</string>
              </property>
             </widget>
             <widget class="QLabel" name="labelStatusAds1248">
              <property name="geometry">
               <rect>
                <x>240</x>
                <y>110</y>
                <width>221</width>
                <height>41</height>
               </rect>
              </property>
              <property name="font">
               <font>
                <pointsize>18</pointsize>
                <weight>50</weight>
                <bold>false</bold>
               </font>
              </property>
              <property name="text">
               <string>Detection ongoing...</string>
              </property>
             </widget>
             <widget class="QLabel" name="labelStatusDrv595Info1">
              <property name="geometry">
               <rect>
                <x>240</x>
                <y>240</y>
                <width>200</width>
                <height>41</height>
               </rect>
              </property>
              <property name="font">
               <font>
                <pointsize>18</pointsize>
                <weight>50</weight>
                <bold>false</bold>
               </font>
              </property>
              <property name="text">
               <string>DRV595</string>
              </property>
             </widget>
             <widget class="QLabel" name="labelStatusDrv595">
              <property name="geometry">
               <rect>
                <x>240</x>
                <y>300</y>
                <width>221</width>
                <height>41</height>
               </rect>
              </property>
              <property name="font">
               <font>
                <pointsize>18</pointsize>
                <weight>50</weight>
                <bold>false</bold>
               </font>
              </property>
              <property name="text">
               <string>Detection ongoing...</string>
              </property>
             </widget>
             <widget class="QLabel" name="labelStatusDrv595Info2">
              <property name="geometry">
               <rect>
                <x>240</x>
                <y>270</y>
                <width>111</width>
                <height>41</height>
               </rect>
              </property>
              <property name="font">
               <font>
                <pointsize>18</pointsize>
                <weight>50</weight>
                <bold>false</bold>
               </font>
              </property>
              <property name="text">
               <string>Status:</string>
              </property>
             </widget>
             <widget class="QLabel" name="labelStatusLmp90100CsInfo1">
              <property name="geometry">
               <rect>
                <x>240</x>
                <y>430</y>
                <width>200</width>
                <height>41</height>
               </rect>
              </property>
              <property name="font">
               <font>
                <pointsize>18</pointsize>
                <weight>50</weight>
                <bold>false</bold>
               </font>
              </property>
              <property name="text">
               <string>LMP90100 CS</string>
              </property>
             </widget>
             <widget class="QLabel" name="labelStatusLmp90100Cs">
              <property name="geometry">
               <rect>
                <x>240</x>
                <y>490</y>
                <width>221</width>
                <height>41</height>
               </rect>
              </property>
              <property name="font">
               <font>
                <pointsize>18</pointsize>
                <weight>50</weight>
                <bold>false</bold>
               </font>
              </property>
              <property name="text">
               <string>Detection ongoing...</string>
              </property>
             </widget>
             <widget class="QLabel" name="labelStatusLmp90100CsInfo2">
              <property name="geometry">
               <rect>
                <x>240</x>
                <y>460</y>
                <width>111</width>
                <height>41</height>
               </rect>
              </property>
              <property name="font">
               <font>
                <pointsize>18</pointsize>
                <weight>50</weight>
                <bold>false</bold>
               </font>
              </property>
              <property name="text">
               <string>Status:</string>
              </property>
             </widget>
             <widget class="QLabel" name="labelStatusMcp4716Info2">
              <property name="geometry">
               <rect>
                <x>700</x>
                <y>270</y>
                <width>111</width>
                <height>41</height>
               </rect>
              </property>
              <property name="font">
               <font>
                <pointsize>18</pointsize>
                <weight>50</weight>
                <bold>false</bold>
               </font>
              </property>
              <property name="text">
               <string>Status:</string>
              </property>
             </widget>
             <widget class="QLabel" name="label_3">
              <property name="geometry">
               <rect>
                <x>490</x>
                <y>220</y>
                <width>161</width>
                <height>140</height>
               </rect>
              </property>
              <property name="text">
               <string/>
              </property>
              <property name="pixmap">
               <pixmap resource="qdarkstyle/style.qrc">:/qss_icons/Images/MC4716.png</pixmap>
              </property>
             </widget>
             <widget class="QLabel" name="labelStatusStm32f401reInfo1">
              <property name="geometry">
               <rect>
                <x>700</x>
                <y>430</y>
                <width>221</width>
                <height>41</height>
               </rect>
              </property>
              <property name="font">
               <font>
                <pointsize>18</pointsize>
                <weight>50</weight>
                <bold>false</bold>
               </font>
              </property>
              <property name="text">
               <string>STM32F4 NUCLEO</string>
              </property>
             </widget>
             <widget class="QLabel" name="labelStatusStm32f401reInfo2">
              <property name="geometry">
               <rect>
                <x>700</x>
                <y>460</y>
                <width>111</width>
                <height>41</height>
               </rect>
              </property>
              <property name="font">
               <font>
                <pointsize>18</pointsize>
                <weight>50</weight>
                <bold>false</bold>
               </font>
              </property>
              <property name="text">
               <string>Status:</string>
              </property>
             </widget>
             <widget class="QLabel" name="labelStatusLmp90100SmInfo1">
              <property name="geometry">
               <rect>
                <x>700</x>
                <y>50</y>
                <width>200</width>
                <height>41</height>
               </rect>
              </property>
              <property name="font">
               <font>
                <pointsize>18</pointsize>
                <weight>50</weight>
                <bold>false</bold>
               </font>
              </property>
              <property name="text">
               <string>LMP90100 SM</string>
              </property>
             </widget>
             <widget class="QLabel" name="labelStatusLmp90100Sm">
              <property name="geometry">
               <rect>
                <x>700</x>
                <y>110</y>
                <width>221</width>
                <height>41</height>
               </rect>
              </property>
              <property name="font">
               <font>
                <pointsize>18</pointsize>
                <weight>50</weight>
                <bold>false</bold>
               </font>
              </property>
              <property name="text">
               <string>Detection ongoing...</string>
              </property>
             </widget>
             <widget class="QLabel" name="label_5">
              <property name="geometry">
               <rect>
                <x>480</x>
                <y>410</y>
                <width>171</width>
                <height>140</height>
               </rect>
              </property>
              <property name="text">
               <string/>
              </property>
              <property name="pixmap">
               <pixmap resource="qdarkstyle/style.qrc">:/qss_icons/Images/Nucleo.gif</pixmap>
              </property>
             </widget>
             <widget class="QLabel" name="labelStatusMcp4716">
              <property name="geometry">
               <rect>
                <x>700</x>
                <y>300</y>
                <width>221</width>
                <height>41</height>
               </rect>
              </property>
              <property name="font">
               <font>
                <pointsize>18</pointsize>
                <weight>50</weight>
                <bold>false</bold>
               </font>
              </property>
              <property name="text">
               <string>Detection ongoing...</string>
              </property>
             </widget>
             <widget class="QLabel" name="label_6">
              <property name="geometry">
               <rect>
                <x>470</x>
                <y>30</y>
                <width>210</width>
                <height>140</height>
               </rect>
              </property>
              <property name="text">
               <string/>
              </property>
              <property name="pixmap">
               <pixmap resource="qdarkstyle/style.qrc">:/qss_icons/Images/LMP90100.gif</pixmap>
              </property>
             </widget>
             <widget class="QLabel" name="labelStatusLmp90100SmInfo2">
              <property name="geometry">
               <rect>
                <x>700</x>
                <y>80</y>
                <width>111</width>
                <height>41</height>
               </rect>
              </property>
              <property name="font">
               <font>
                <pointsize>18</pointsize>
                <weight>50</weight>
                <bold>false</bold>
               </font>
              </property>
              <property name="text">
               <string>Status:</string>
              </property>
             </widget>
             <widget class="QLabel" name="labelStatusStm32f401re">
              <property name="geometry">
               <rect>
                <x>700</x>
                <y>490</y>
                <width>221</width>
                <height>41</height>
               </rect>
              </property>
              <property name="font">
               <font>
                <pointsize>18</pointsize>
                <weight>50</weight>
                <bold>false</bold>
               </font>
              </property>
              <property name="text">
               <string>Detection ongoing...</string>
              </property>
             </widget>
             <widget class="QLabel" name="labelStatusMcp4716Info1">
              <property name="geometry">
               <rect>
                <x>700</x>
                <y>240</y>
                <width>200</width>
                <height>41</height>
               </rect>
              </property>
              <property name="font">
               <font>
                <pointsize>18</pointsize>
                <weight>50</weight>
                <bold>false</bold>
               </font>
              </property>
              <property name="text">
               <string>MCP4716</string>
              </property>
             </widget>
            </widget>
            <widget class="QFrame" name="frame_3">
             <property name="geometry">
              <rect>
               <x>940</x>
               <y>9</y>
               <width>930</width>
               <height>615</height>
              </rect>
             </property>
             <property name="minimumSize">
              <size>
               <width>0</width>
               <height>100</height>
              </size>
             </property>
             <property name="frameShape">
              <enum>QFrame::Box</enum>
             </property>
             <property name="frameShadow">
              <enum>QFrame::Plain</enum>
             </property>
             <property name="lineWidth">
              <number>3</number>
             </property>
            </widget>
           </widget>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="tab">
         <attribute name="title">
          <string>Page</string>
         </attribute>
        </widget>
        <widget class="QWidget" name="tab_2">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <attribute name="title">
          <string>Tab 2</string>
         </attribute>
         <layout class="QGridLayout" name="gridLayout_2"/>
        </widget>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QDockWidget" name="dscDataDockWidget">
    <property name="geometry">
     <rect>
      <x>1580</x>
      <y>720</y>
      <width>340</width>
      <height>281</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="contextMenuPolicy">
     <enum>Qt::NoContextMenu</enum>
    </property>
    <property name="windowTitle">
     <string>DSC / Device Data</string>
    </property>
    <widget class="QWidget" name="dockWidgetContents_5">
     <layout class="QGridLayout" name="gridLayout_8"/>
    </widget>
   </widget>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1920</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="qdarkstyle/style.qrc"/>
 </resources>
 <connections/>
 <slots>
  <slot>ButtonClicked()</slot>
 </slots>
</ui>
